mne/viz/evoked.py

```diff
@@ -1118,8 +1119,9 @@ def whitened_gfp(x, rank=None):
         titles_['meg'] = 'MEG (combined)'
 
     colors = [plt.cm.Set1(i) for i in np.linspace(0, 0.5, len(noise_cov))]
+    # XXX : should maybe use _handle_default with color arg...
```

Comment:
any reason you didn't put it in? did you try and it didn't work / wouldn't be trivial? No need to add `color` arg for now, but a `_handle_default(..., None)` would be good here

Reply:
I marked it for later.